dev-infrastructure/modules/aks-cluster-base.bicep

```
@@ -103,9 +104,8 @@ var systemAgentPool = [
   }
 ]
 
-var userAgentPool = [
-  {
-    name: 'user'
+var userAgentPool = [for i in range(0, userAgentPoolAZCount): {
+    name: 'user-${(i + 1)}'
```

```suggestion
    name: 'user${(i + 1)}'
```
The name of a node pool may only contain lowercase alphanumeric characters and must begin with a lowercase letter. For Linux node pools, the length must be between 1-12 characters. (Hyphen is not allowed)